spec/vendor_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+
+RSpec.describe Vendor do
+  before(:each) do
+    @vendor = Vendor.new("Rocky Mountain Fresh")
+    @vendor1 = Vendor.new("Rocky Mountain Fresh")
+    @vendor2 = Vendor.new("Ba-Nom-a-Nom")
+    @vendor3 = Vendor.new("Palisade Peach Shack")    
+    @item1 = Item.new({name: 'Peach', price: "$0.75"})
+    @item2 = Item.new({name: 'Tomato', price: '$0.50'})
+    @item3 = Item.new({name: "Peach-Raspberry Nice Cream", price: "$5.30"})
+    @item4 = Item.new({name: "Banana Nice Cream", price: "$4.25"})
+  end
+
+  it 'can initialize' do
+    expect(@vendor).to be_instance_of(Vendor)
+  end
+
+  it 'has a name' do
+    expect(@vendor.name).to eq("Rocky Mountain Fresh")
+  end
+
+  it 'starts with an empty inventory' do
+    expect(@vendor.inventory).to eq({})
+  end
+
+  it 'can check stock' do
+    expect(@vendor.check_stock(@item1)).to eq(0)
+  end
+
+  it 'can stock items' do
+    @vendor.stock(@item1, 30)
```

Comment:
A good test here would be to stock the same items a few times, and check that the quantity of your item is accumulating